go/imageio/backend.go

```diff
@@ -1,10 +1,5 @@
-// ovirt-imageio
-// Copyright (C) 2021 Red Hat, Inc.
-//
-// This program is free software; you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as
```

Comment:
The go/imageio directory is a library using the `GNU Lesser General Public License` (LPGL).